src/components/molecules/Schedule/Schedule.types.ts

```diff
@@ -0,0 +1,19 @@
+import { Room } from "types/rooms";
+import { PersonalizedVenueEvent } from "types/venues";
+import { WithVenueId } from "utils/id";
+
+export type RoomWithEvents = Room & {
+  events: WithVenueId<PersonalizedVenueEvent>[];
+};
+
+export interface ScheduleDay {
+  isToday: boolean;
+  weekday: string;
```

Comment:
I think this could be defined in a stricter way?

Reply:
@mike-lvov What do you mean by 'in a stricter way'?

Reply:
I guess weekday can have only 7 values: eg. Monday, Tuesday, Wednesday..... If so, it should be defined as that.

Reply:
for getting `weekday` I use `format()` function (https://date-fns.org/docs/format) which return type is `string`, that means that I'll need explicitly specify the type when I assign weekday

like here: 
```js
export type Weekday = 'Mon' | 'Tue' | 'Wed'  .....

export interface ScheduleDay {
  isToday: boolean;
  weekday: Weekday;
  dayStartUtcSeconds: number;
  locatedEvents: LocatedEvents[];
  personalEvents: WithVenueId<PersonalizedVenueEvent>[];
}

//....
   weekday: format(dayStart, "E") as Weekday,
```

so is that okay?

Reply:
> that means that I'll need explicitly specify the type when I assign weekday

I don't believe that is true. A `string` with the value `Mon` will satisfy that `Weekday` type without needing the `as Weekday`

Reply:
removed `weekday` property as we don't need remember it: https://github.com/sparkletown/sparkle/pull/1302/commits/6d26a6f4f896aa3cc48d0eafadb84c9807e4c24d